lib/runner.js

```diff
@@ -157,15 +143,21 @@ var expandArchetype = function (cmd, opts, env) {
  * @returns {Object}            Child process object
  */
 var run = function (cmd, shOpts, opts, callback) {
-  var maxBuffer = opts.unlimitedBuffer ? Infinity : MAX_BUFFER;
+  var sh = "sh";
+  var shFlag = "-c";
 
   // Update shell options and ensure basic structure.
   shOpts = _.extend({
-    maxBuffer: maxBuffer,
-    env: {}
+    env: {},
+    stdio: "inherit"
   }, shOpts);
 
-  var buffer = opts.buffer;
+  if (process.platform === "win32") {
```

Comment:
NIT: Is there a weblink for where you got this wrapper code? Or maybe just more explanation about windows shell if you're doing it from scratch?